frontend/cypress/support/commands.ts

```diff
@@ -26,32 +28,38 @@ Cypress.Commands.add("checkCurrentActivePage", (activePageNumber) => {
 })
 
 Cypress.Commands.add("mockGetRelatedTags", () => {
-    cy.intercept("GET", `${apiBaseUrl}/api/query/v1/tags/related`, {
+    cy.intercept("GET", `${apiBaseUrlMock}/api/query/v1/tags/related`, {
         fixture: "tags/related.json"
     }).as("mockGetRelatedTags")
 })
 
 Cypress.Commands.add("mockItemsList", (itemCount?: number) => {
-    const body = itemCount || itemCount === 0 ? items.slice(0, itemCount) : items
+    const body = itemCount || itemCount === 0 ? {
```

Comment:
This is not correct. Check api https://query.wwweights.com/swagger#/default/ItemsController_getItemList